core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -84,7 +84,7 @@ import scala.jdk.CollectionConverters._
 class ReplicaManagerTest {
 
   val topic = "test-topic"
-  val topicId = Uuid.randomUuid()
+  val topicId: Uuid = Uuid.randomUuid()
```

Comment:
did we need this?

Reply:
It is surprising that we only have to do this for `topicId`.  Instead of doing this, I wonder if we should make all the attributes private. What do you think?